app/src/main/java/com/github/sdp/mediato/ui/BaseProfileFragment.java

```diff
@@ -109,11 +107,19 @@ private void observeFollowingAndFollowersCount() {
   }
 
   private void setupFollowingButton(Button followingButton) {
-    followingButton.setOnClickListener(v -> switchFragment(new MyFollowingFragment()));
+    followingButton.setOnClickListener(v -> {
```

Comment:
What was the issue with switchFragment here ? I didn't notice something not functioning properly when using the app.

Reply:
I changed it because there is an interface `FragmenSwitcher` specifically made for switching fragments that is used in other parts of the code. It calls a switch fragment method already defined in the MainActivity. So this switchFragment method here was duplicated code. For some reason the following and follower fragments crashed though and I had to pass the arguments again.

Reply:
Okay thanks !